opensearch_py_ml/ml_models/sentencetransformermodel.py

```diff
@@ -1006,6 +1008,55 @@ def set_up_accelerate_config(
                 "Failed to open config file for ml common upload: " + file_path + "\n"
             )
 
+    def _get_model_description_from_md_file(self, readme_file_path) -> str:
```

Comment:
Fromt the comment below, 
> Get description of the model from README.md file

Seems we should rename the method name as `_get_model_description_from_readme_file`

Reply:
Sure.